.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!-- This is only a suggestion of topics to cover on your PR description -->
+<!-- Feel free to ignore it or remove irrelevant sections -->
+
+## How does this PR change the system?
+
+<!-- How and why this PR changes the system -->
```

Comment:
I've found it really useful to concretely separate this a bit: first sentence or para is *why* (what is the problem, what do we want to make better). Then, *what*: a "guide" to the PR for a reviewer, often in bullet points, with the overall changes you tried to make. This should include any factoring as well; it is here to give the reviewer context to understand the +'s and -'s in the PR.